desktop/apps/artwork/components/artists/templates/biography.jade

```diff
@@ -14,7 +14,10 @@
   .artwork-artist__content__short-bio
     p= artist.bio.replace('born', 'b.')
 
-if artist.exhibition_highlights && artist.exhibition_highlights.length <= 15
-  include ./exhibition
-else if artist.exhibition_highlights && artist.exhibition_highlights.length > 15
-  include ./highlights
+div(id="market-insights-container-" + artist._id)
```

Comment:
Since an artwork can have multiple artists, each artist section (below the artwork), can have its own div for market insights (unique id).